### DISCUSSION TITLE
[RELEASE] Tram

### DISCUSSION THREAD

**Discussion by qloakongithub** [nzp-team/nzportable#1057] [Map Releases]
As a new map maker for NZP, this is my first map! It is set in a tram, with 2 windows, a Pack-a-Punch machine, a Mystery Box, Quick Revive, and a random Perk! This map is based on the ending of Half-Life 1; where you are given a choice between becoming an employee of G-Man, or being teleported to an alien world to die. In this map, instead of being teleported, you (Gordon Freeman) are left on the tram to fight off hoards of zombies surrounding you. Your goal; survive as long as you can.

Version 1.0: Original map version

 [DOWNLOAD v1.0](https://archive.org/download/tram_nzp/Tram.zip)

Version 1.1: Fixes issues, and adds Spare Change to the Perk machines. 

 [DOWNLOAD v1.1](https://archive.org/download/tram-v-1.1/Tram_v1.1.zip)

![fte-20241222000148-0](https://github.com/user-attachments/assets/50d1abae-c403-4e57-b069-22ccebfd3479)

![fte-20241222000152-0](https://github.com/user-attachments/assets/285e36c4-2bdb-408c-8778-eba7aaadac03)

**Comment by mmkkiinn** [comment]
This a good map but change your Idea to quick revive and juggernaut

**Reply by qloakongithub** [reply]
I thought of that but I wanted people to have to be able to survive with what they have, whether that be Juggernog, Double Tap, or whatever. Thank you!

**Comment by TheIdioticGamer** [comment]
Why does it remind me of the Tram in the very beginning of Half-Life 1?

**Reply by qloakongithub** [reply]
That's what it was inspired by lol.
